web/language/management/commands/get_categories.py

```diff
@@ -0,0 +1,26 @@
+from django.core.management.base import BaseCommand, CommandError
+
+import psycopg2
+
+class Command(BaseCommand):
+
+    # Get the local database address passed in from the bash script
+    def add_arguments(self, parser):
+            parser.add_argument('hostAddress')
+
+    def handle(self, *args, **options):
+        hostAddress = options['hostAddress']
+
+        # Connect to the database
+        conn = psycopg2.connect(host=hostAddress, dbname="postgres", user="postgres")
+        cur = conn.cursor()
+
+        # Open the csv and copy it into the database row by row
+        with open('./fixtures/categories.csv', 'r') as f:
+            next(f)
+
+            cur.copy_from(f, 'language_placenamecategory', sep=',')
+
```

Comment:
@NolanVH you might want to consider using the Django ORM like the other scripts.

from language.models import Category
category.objects.create(dict(f)**) # f needs converted to a dict with the right keys etc.

Reply:
@countable I have made the changes as suggested and added them to the pull request. Thanks for the review!